src/pudl/analysis/record_linkage/eia_ferc1_record_linkage.py

```diff
@@ -499,25 +590,16 @@ def run_model(
             "l1_ratio": [0.1, 0.3, 0.5, 0.7, 0.9],
         },
     ]
-    x_train = features_train.to_numpy()
-    y_train = np.where(
-        features_train.merge(
-            train_df,
-            how="left",
-            left_index=True,
-            right_index=True,
-            indicator=True,
-        )["_merge"]
-        == "both",
-        1,
-        0,
+    X = features_train.matrix  # noqa: N806
+    X_train, X_test, y_train, y_test = train_test_split(  # noqa: N806
+        X, y_df, test_size=0.25, random_state=16
```

Comment:
I added a train and test split, since previously accuracy was just being measured on the train set. Accuracy on the test set is .74 and the results of grid search cross validation on the train set are also outputted (.94)